### PR TITLE
Adds smg to replace the mosin as nuke op surplus gun. Remake of #51285

### PR DESCRIPTION
**Pull request by ghost**
i managed to completely fuck over the last branch

## About The Pull Request
Adds the PP-95, an smg to replace the mosin as a nuclear operative's surplus weapon.

image

## Why It's Good For The Game
The surplus rifle at the moment is a 1tc gun with bullets that do as much damage as a point blank bulldog shotgun with buckshot. I don't think a surplus gun that cheap should be able to kill that fast, but maybe it's just me. This replaces the mosin with a gun actually worth what you paid for.

## Changelog
:cl:
add: PP-95 smg and it's magazine
tweak: replaced mosin with pp-95 in the nuke op uplink
/:cl:

### CODE REVIEW

**Comment by Paxilmaniac** [comment]
fuck i forgot the inhands
the entire reason i redid it

**Comment by MrDoomBringer** [comment]
image
hmm yes

unironically your last PR got a lot better after the damage increase. My only 2 cents is that making it wildly inaccurate would be funny as hell
👍  either way tho

**Comment by MrDoomBringer** [comment]
also i like the bison aesthetically for some reason so ig i'm biased

**Comment by optimumtact** [comment]
kind of cringe

**Comment by MrDoomBringer** [comment]
@optimumtact the pink haired girls are having a conversation?? can you come back later

**Comment by optimumtact** [comment]
I meant this PR

**Comment by MrDoomBringer** [comment]
i see, yeah that's understandable

**Comment by necromanceranne** [comment]
Wait are you sure those inhands for the arg are correct?

**Comment by Paxilmaniac** [comment]
probably not, ill check

**Comment by Paxilmaniac** [comment]
bruh

File: code/modules/projectiles/boxes_magazines/external/smg.dm [hunk 1]
```diff
@@ -27,6 +27,20 @@
...
 
+/obj/item/ammo_box/magazine/plastikov9mm
+	name = "PP-95 magazine (9x19mm)"
+	icon_state = "9x19-50"
+	ammo_type = /obj/item/ammo_casing/c9x19mm
+	caliber = "9x19mm"
+	max_ammo = 50
+
+/obj/item/ammo_box/magazine/plastikov9mm/update_icon()
+	..()
```
Comment (Rohesie) [inline code/modules/projectiles/boxes_magazines/external/smg.dm:38]:
```suggestion
	. = ..()
```

**Comment by Rohesie** [comment]
@Twaticus , are you okay with these sprites?

Another gun added to the arsenal. I don't care either way much, but if nobody denies it I'll end up merging it.

**Review by Twaticus — CHANGES_REQUESTED** [review]
looks like the righthand north and south sprites need to be shifted, and the righthand side sprites are using the lefthand version instead of their own

e
(lefthand vs righthand)

**Comment by Paxilmaniac** [comment]
Fuck.

**Review by Twaticus — APPROVED** [review]

**Pull request merged**